src/github/repository.ts

```diff
@@ -54,13 +52,12 @@ export const deleteRepositoryWebhookHandler = async (context: WebhookContext, ji
 	context.log.info(`Deleting dev info for repo ${context.payload.repository?.id}`);
 
 	try {
-		const jiraResponse = await jiraClient.devinfo.repository.delete(
-			transformRepositoryId(context.payload.repository?.id, context.gitHubAppConfig?.gitHubBaseUrl)
-		);
+		const jiraResponse =  await jiraClient.devinfo.installation.delete(gitHubInstallationId);
```

Comment:
Man, implemented this so differently to start with but then simplified... and then simplified again when I discovered we already have installation delete and were just calling the wrong endpoint 🤦